frontend/src/lib/components/AdminPage.svelte

```diff
@@ -21,7 +21,6 @@ onMount(async () => {
 </script>
 
 <Tabs tabStyle="underline" class="w-full">
```

Comment:
in line 30, there is the `MilestoneGroups` component loaded into the tabitem. I'm a little confused by this distinction now. Is this correct, or rather, is the `Milestone` part missing here?